assets/scss/components/stencil/cart/_cart.scss

```diff
@@ -20,6 +20,8 @@ $cart-thumbnail-paddingVertical:        0.5rem;
 $cart-item-label-offset:                $cart-thumbnail-maxWidth + $cart-item-spacing;
 $card-preview-zoom-bottom-offset:       6rem;
 
+$previewCart-additionalCheckoutButtons-maxWidth: 274px;
```

Comment:
Could you please explain why 274px value has been choosen?

Reply:
@BC-tymurbiedukhin  I chose this size based on the size of the same buttons at a minimum screen extension of 320. This is done to make these buttons look similar.
<img width="193" alt="465_min-width_btn" src="https://user-images.githubusercontent.com/82589781/120788168-d4905000-c538-11eb-93c5-0691232dd6df.png">

Reply:
@BC-tymurbiedukhin I think I found a better option for setting the maximum width for buttons using padding. There is no need to use specific numbers for the maximum width and create a new variable for it.

<img width="425" alt="465_max-width_btns" src="https://user-images.githubusercontent.com/82589781/121169985-4b8d5780-c85d-11eb-9439-9e2a3a877c93.png">